Resources/input/fsh/ISiKAbrechnungsfall.fsh

```diff
@@ -1,35 +1,110 @@
+  * ^short = "Abrechnungsdiagnose /-prozedur"
+  * ^comment = "Insbesondere bei Abrechnungen im DRG-Kontext muss eine Diagnose als Hauptdiagnose und 
+  ggf. weitere Diagnosen als abrechnungsrelevante Nebendiagnosen klassifiziert werden. Diese Extension ermöglicht es, diese Qualifikation im Abrechnungskontext vorzunehmen, 
+  unabhängig von der *medizinischen* Relevanz, die in `Encounter.diagnosis` erfolgt."
+  * extension[Use] 1..1 MS
+  * extension[Referenz] 1..1 MS
 * identifier 1.. MS
   * ^slicing.discriminator.type = #pattern
   * ^slicing.discriminator.path = "$this"
   * ^slicing.rules = #open
 * identifier contains Abrechnungsnummer 1..1 MS
 * identifier[Abrechnungsnummer] only IdentifierAbrechnungsnummer
+  * ^short = "Abrechnungsfallnummer"
+  * ^comment = "Im DRG-Kontext werden häufig sämtliche Besuche (`Encounter`), die unter einen gemeinsamen Abrechnungskontext zusammengefasst werden, 
+  unter einer &quot;Fallnummer&quot; geführt. In dieser Konstellation sind die Begriffe &quot;Fallnummer&quot; und &quot;Abrechnungsfallnummer&quot; gleichbedeutend.  
+  Dies ist insbesondere im Kontext des Mappings zwischen HL7 V2 und HL7 FHIR zu beachten, da es in V2 Usus ist, 
+  die Fallnummer (eigentlich Identifier des Abrechnungsfalles) im `PV1`-Segment (Patient Visit) zu übermitteln. 
+  Es handelt sich dabei jedoch *nicht* um den Identifier des Besuchs (`Encounter`) sondern den des Abrechnungsfalles (`Account`), 
+  da der Identifier oft für die Gruppierung mehrerer Besuche (z.B. vorstationär + stationär + nachstationär) mit gemeinsamem (DRG)-Kontext verwendet wird.  
+  Die Abrechnungsfallnummer in `Account.identifier` muss identisch sein mit dem Identifier, 
+  der bei den Encountern, die unter diesem Account gruppiert werden, unter `Encounter.account.identifier` angegeben ist."
   * ^patternIdentifier.type = $v2-0203#AN
   * type MS
+    * coding
+      * ^short = "Codierte Darstellung des Identifier-Typs"
+      * system 1.. MS
+        * ^short = "Codier-Schema"
+        * ^comment = "Hier ist stets der Wert `http://terminology.hl7.org/CodeSystem/v2-0203` anzugeben." 
+      * code 1.. MS
+        * ^short = "Code"
+        * ^comment = "Hier ist stets der Wert `AN` anzugeben."
   * system MS
+    * ^short = "Namensraum des Identifiers"
+    * ^comment = "Hier ist stets der eindeutige Name (URL) des Namensraums anzugeben, 
+    aus dem der Identifier stammt. 
+    Hinweise zur Festlegung der URLs für lokale Namensräume sind in den 
+    [Deutschen Basisprofilen](https://simplifier.net/guide/leitfaden-de-basis-r4/ig-markdown-Terminologie-Namensraeume?version=current) beschrieben.  
+    **Begründung Pflichtfeld:** `system` stellt in Kombination mit `value` die Eindeutigkeit eines Identifiers sicher."
   * value MS
+    * ^comment = "Enthält den eigentlichen Wert des Identifiers.  
+    **Begründung Pflichtfeld:** Ist der Wert nicht bekannt, sollte der gesamte Slice weggelassen werden."
+
 * status MS
-* type 1.. MS
```

Comment:
question: Warum MS auf type entfernt?

ursprünglich galt:

>  Bedeutung: Alle Accounts MÜSSEN eindeutig kategorisiert werden können, sodass eine Filterung pro Typ möglich ist.
> 
> Hinweise: Verpflichtende Kodierung per [HL7 v3 Value Set ActEncounterCode](https://hl7.org/fhir/R4/v3/ActEncounterCode/vs.html) als Äquivalent zu Encounter.class.

(https://simplifier.net/guide/isik-basis-v4/ImplementationGuide-markdown-Datenobjekte-Datenobjekte_Abrechnungsfall?version=current)

Reply:
wegen https://service.gematik.de/browse/PTDATA-1236 
Wir hatten dazu einen gemeinsamen Termin in dem wir entschieden hatten, das MS wegzunehmen, da derzeit gar nicht klar ist, was der Sinn und Zeck von Account.type ist.